module-2-html/exercises/02-book/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="author" content="Tim Berners-Lee aka Bruce Lee">
+        <meta name="description" content="Markup languages book">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>A book about markup languages</title>
+    </head>
+    <body>
+        <h1>1. Markup languages</h1><br>
+        <p>A <strong>markup language</strong> is a computer language that uses <em>tags</em> to define <em>elements</em> within a document. 
+            Markup languages are used to <em>annotate</em> a document in a way that is syntactically distinguishable from the text, 
+            and can assign <em>semantic meaning</em> to the text.</p>
+        <h3>Types of markup languages</h3>

```

Comment:
Ako bi stavljao naslov ovdje morao bi koristiti `<h2>` jer je prema hijerarhiji naslova u dokumentu `<h1>` jedini roditelj. Međutim, u sadržaju nisam naznačio da je to nekakav oblik poglavlja / potpoglavlja, pa naslov recimo ne bih ovdje ni stavljao :D

Ako želiš istaknuti sadržaj u ovom slučaju, `<strong>` ne bi bio loš izbor.

Reply:
Malo me zavaralo što nema točke na kraju rečenice, pa sam to shvatio kao nekakav naslov... Ali kužim poantu :)

Reply:
Hoću reći nema dvotočke... :)